debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -199,15 +224,17 @@
                                                      DATABASE_LIST,
                                                      COLLECTION_WHITELIST,
                                                      COLLECTION_BLACKLIST,
-                                                     AUTO_DISCOVER_MEMBERS);
+                                                     AUTO_DISCOVER_MEMBERS,
+                                                     DATABASE_WHITELIST,
+                                                     DATABASE_BLACKLIST);
 
     protected static Field.Set EXPOSED_FIELDS = ALL_FIELDS;
 
     protected static ConfigDef configDef() {
         ConfigDef config = new ConfigDef();
         Field.group(config, "MongoDB", HOSTS, USER, PASSWORD, LOGICAL_NAME, CONNECT_BACKOFF_INITIAL_DELAY_MS,
                     CONNECT_BACKOFF_MAX_DELAY_MS, MAX_FAILED_CONNECTIONS, AUTO_DISCOVER_MEMBERS);
-        Field.group(config, "Events", DATABASE_LIST, COLLECTION_WHITELIST, COLLECTION_BLACKLIST);
+        Field.group(config, "Events", DATABASE_LIST, DATABASE_WHITELIST, DATABASE_BLACKLIST, COLLECTION_WHITELIST, COLLECTION_BLACKLIST);
```

Comment:
Hum, so that's a bit weird, there already was `DATABASE_LIST` before, but just its definition, I can't seem to find code that would read that property, also it's not part of the docs. Seems like an uncompleted attempt of adding an option for specifying the DBs. So I'd suggest to remove this with this PR as well, as it's superseded by the new options now.